packages/one-app-server-bundler/utils/patchedCryptoHash.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2023 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+const crypto = require('crypto');
+
+// Monkey Patch for unsupported hash algo. Needed to support Node >=17.
+// https://github.com/webpack/webpack/issues/13572#issuecomment-923736472
+const originalCreateHash = crypto.createHash;
+crypto.createHash = (algo) => originalCreateHash(algo === 'md4' ? 'sha256' : algo);
```

Comment:
may not need as it should have been fixed in webpack 5.
https://github.com/webpack/webpack/pull/14306

webpack 5.54.0 it was tagged

Reply:
Oh I see, the ides is that we patched this just so that one plugin would behave, but now it does by default 🤔 

let me test it out a little, and see what that plugin does

Reply:
Looks like I _can_ remove this, for one-app-bundler.

But to note, you have comment this on one-app-server-bundler. Which is still v4, so I cant removed this from _here_

Reply:
done in [aa2c5a5](https://github.com/americanexpress/one-app-cli/pull/568/commits/aa2c5a5ad7f279996b6ab733f777a43ccba0eb8b)